students/gotem888/hw-12 searchEngine/css/style.css

```diff
@@ -0,0 +1,276 @@
+@property --rotate {
+    syntax: "<angle>";
+    initial-value: 132deg;
+    inherits: false;
+}
+:root {
+    --form-height: 40vh;
+    --form-width: calc(var(--form-height) / 1.5);
+    --blue: #3498db;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    margin: 0;
```

Comment:
You don't need `margin: 0` as you already have it in `reset.css` file

Reply:
<img width="435" alt="Screen Shot 2022-02-05 at 22 59 16" src="https://user-images.githubusercontent.com/42999342/152658870-10a36b16-2d14-4db4-9ad4-cfd1a69bfcda.png">